src/ostorlab/runtimes/local/app/common.py

```diff
@@ -0,0 +1,8 @@
+"""common utilities for the flask app."""
+
+import graphene
+
+
+class SortEnum(graphene.Enum):
```

Comment:
Since the values in the enum type are constant, I would represent them in UPPERCASE letters,

Fell free to check this: https://docs.python.org/3/howto/enum.html#:~:text=Because%20Enums%20are%20used%20to,that%20style%20in%20our%20examples.

Reply:
We are using this in RE also, for consistency i prefer keep it this way.

Reply:
Hamza is right this is a separate project please update the enum